content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+# Node Types
+
+## cloudify.gcp.nodes.Address
+**Derived From:** cloudify.gcp.nodes.GlobalAddress
+
+A GCP Address. This can be connected to an Instance using the `cloudify.gcp.relationships.instance_connected_to_ip` relationship type
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `region`
+    Region to place the Address in. If not provided it defaults to the value in `gcp_config` (which defaults to 'default').
+
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.BackendService
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A group of Instances (contained within InstanceGroups) which can be used as the backend for load balancing.
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Optional additional settings. Possible fields in dictionary are: port_name, protocol, timeout_sec.
+
+    *default:* {}
+  * `health_check`
+    URL of a health check assigned to this backend service.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional health check name. By default it will be backend service id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.DNSAAAARecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`AAAA` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* AAAA
+
+
+
+
+
+## cloudify.gcp.nodes.DNSMXRecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`MX` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* MX
+
+
+
+
+
+## cloudify.gcp.nodes.DNSNSRecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`NS` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* NS
+
+
+
+
+
+## cloudify.gcp.nodes.DNSRecord
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+Corresponds to a particular subdomain (or `@` for the root) and record-type in the containing DNSZone.
+
+e.g. the `A` record for `special_service.getcloudify.org`
+
+A number of convenience types are provided which update the default type (see DNSAAAARecord, DNSMXRecord, DNSTXTRecord, DNSNSRecord)
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `type`
+    The type of this DNS record. Only one record of each type is allowed with the same name within a zone.
+
+    *type:* string
+    *default:* A
+  * `name`
+    The subdomain. This will be prepended to the DNSZone's dns_name to produce the full domain name for this record. Defaults to the instance ID.
```

Comment:
the instance ~~ID~~  **id**

(Changed for consistency with how it is written elsewhere in the plugin. )